builder/dispatchers.go

```diff
@@ -47,6 +47,22 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 		return fmt.Errorf("Bad input to ENV, too many args")
 	}
 
+	// NOT USED
+	// Just here to show how to use the builder flags stuff within the
+	// context of a builder command. Will remove once we actually add
```

Comment:
Might want to mention TODO/FIXME for easier discovery?
(just browsing through this PR)